### PR TITLE
Added Windows Google Drive client artifacts.

### PR DESCRIPTION
**Pull request by joachimmetz**
@destijl for review

Note that I'll need to create the forensics wiki page and add Mac OS X at a later date

[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/92)

### CODE REVIEW

File: definitions/cloud_services.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Cloud service artifacts.
+
+name: CloudStorageClients
```
Comment (destijl) [inline definitions/cloud_services.yaml]: This is confusing when there is only one here, lets just create it when we have more than one.

**Comment by destijl** [comment]
LGTM

File: definitions/cloud_services.yaml (hunk 1 above)
Comment (joachimmetz) [inline definitions/cloud_services.yaml]: True, let me see if should add some more ;)

**Comment by joachimmetz** [comment]
@destijl can you have a second look at this I've added dropbox as well to make sure the CloudStorageClients had more than one entry.

**Comment by destijl** [comment]
LGTM but lets hold off on merging this until we can update GRR with the new label, or it will break if people import it into GRR.

**Comment by destijl** [comment]
The new label is present in grr, so this can go in.

**Pull request merged**